quiz.css

```diff
@@ -0,0 +1,91 @@
+body {
+	margin: 0;
+}
+
+/*Center the body*/
+
+.contentContainer {
+	margin: 0 1%;
+}
+
+/*Base card styling*/
+article {
+	border: 1px solid lightblue;
+	width: 30%;
+	display: inline-block;
+	height: 100%;
+	margin-bottom: 7px;
+} 
+
+article:nth-of-type(3n+2) { /* always selects the middle element */
+	margin: 0 4%; /* Equal padding between the elements */
+}
+
+/*Image styles*/
+
+img {
+	display: block;
+	margin: 0 auto;
+	border: 1px solid black;
+}
+
+/* Styles for the card title */
+
+header h1.prodTitle { 
+	text-align: center;
+	border-bottom: 1px dotted lightgrey;
+	margin-top: 0;
+}
+
+header.mainHead {
+	text-align: center;
+
+}
+
+/*Below are the sectionOne styles*/
+
+p.description {
+	min-height: 7em;
+}
+
+section.sectionOne p {
```

Comment:
**Pro tip:** Don't name classes based on the sequence they have in the existing HTML.  What if, next month, your product owner or designer decides to change the order of the sections?  Make the class name reflective of the content that is in an element instead.

Change the name of `sectionOne`, `sectionTwo`, and `sectionThree`